.readthedocs.yaml

```
@@ -0,0 +1,14 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD API version
+version: 2
+
+build:
+  os: ubuntu-lts-latest
+  tools:
+    python: >-
+      3.11
+  commands:
+    - mkdir -p $READTHEDOCS_OUTPUT/html/
+    - cp -r docs/* $READTHEDOCS_OUTPUT/html/
```

Let's make the command outputs verbose + safeguard the env var usage:
```suggestion
  - mkdir -pv "${READTHEDOCS_OUTPUT}"/html/
  - cp -rv docs/* "${READTHEDOCS_OUTPUT}"/html/
```